Modules/Core/Common/test/itkShapedImageNeighborhoodRangeGTest.cxx

```diff
@@ -914,9 +914,11 @@ TEST(ShapedImageNeighborhoodRange, ProvidesReverseIterators)
   const unsigned int numberOfNeighborhoodPixels = 3;
 
   const std::vector<PixelType> stdVector(range.begin(), range.end());
-  std::vector<PixelType>       reversedStdVector1(numberOfNeighborhoodPixels);
-  std::vector<PixelType>       reversedStdVector2(numberOfNeighborhoodPixels);
-  std::vector<PixelType>       reversedStdVector3(numberOfNeighborhoodPixels);
+
+  assert(stdVector.size() == numberOfNeighborhoodPixels);
```

Comment:
Good catch  👍  Please make that `ASSERT_EQ(stdVector.size(), numberOfNeighborhoodPixels)`.  ASSERT_EQ is specific to GoogleTest. It ends the unit test _directly_ at that point, even in Release mode, and reports a test failure.

Reply:
Done!